### PR TITLE
My first goodie

### PR DESCRIPTION
**Pull request by hazeeb**
Just made this first instant answer contribution from #quackandhack meetup bangalore 2015.

What does your Instant Answer do?
It prints my github username and tell that am awesome and completed the goodie hack.

What problem does your Instant Answer solve (Why is it better than organic links)?
It helps to find me among all other github users

What is the data source for your Instant Answer? (Provide a link if possible)
https://duck.co/duckduckhack/goodie_quickstart

Why did you choose this data source?
Because this is my first goodie hack.

Are there any other alternative (better) data sources?

What are some example queries that trigger this Instant Answer?
duckduckhack sagarhani

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
github users

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

Which existing Instant Answers will this one supercede/overlap with?
None

Are you having any problems? Do you need our help with anything?
No

Where did you hear about DuckDuckHack? (For first time contributors)
DuckDuckgo meetup at bangalore,India.

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers).

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Hi @hazeeb, thanks for this! Would you mind filling in the [Goodies PR template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_goodie.md) in your first comment?

File: lib/DDG/Goodie/IsAwesome/hazeeb.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package DDG::Goodie::IsAwesome::hazeeb;
+# ABSTRACT: Write an abstract here
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/hazeeb.pm]: Can you fill out the abstract?

File: lib/DDG/Goodie/IsAwesome/hazeeb.pm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Start at https://duck.co/duckduckhack/goodie_overview if you are new
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/hazeeb.pm]: The rest of these comments can go, they're just for instruction :)

**Comment by moollaza** [comment]
Ping! @hazeeb just checking in! Do you need anything from us? When you're all done implemented the fixed please drop us a comment to let us know!

Thanks :+1:

**Comment by mintsoft** [comment]
@hazeeb Just checking in! Are you interested in following this up? We'd love to see it go live

**Comment by hazeeb** [comment]
What does your Instant Answer do?
It prints my github username and tell that am awesome and completed the goodie hack.

What problem does your Instant Answer solve (Why is it better than organic links)?
It helps to find me among all other github users

What is the data source for your Instant Answer? (Provide a link if possible)
https://duck.co/duckduckhack/goodie_quickstart

Why did you choose this data source?
Because this is my first goodie hack.

Are there any other alternative (better) data sources?

What are some example queries that trigger this Instant Answer?
duckduckhack sagarhani

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
github users

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

Which existing Instant Answers will this one supercede/overlap with?
None

Are you having any problems? Do you need our help with anything?
No

Where did you hear about DuckDuckHack? (For first time contributors)
DuckDuckgo meetup at bangalore,India.

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers).

**Comment by mintsoft** [comment]
@hazeeb are you OK working on the feedback provided?

**Comment by hazeeb** [comment]
@mintsoft Yes.

**Comment by mintsoft** [comment]
Hi @hazeeb Just checking in, are you still interested in finishing this up?

**Comment by hazeeb** [comment]
@mintsoft I'm interested to complete my first instant answer. I've had updated PR template few months back.

**Comment by mintsoft** [comment]
Hi @hazeeb unfortunately we're not accepting any more IsAwesome goodies, so I'm closing this PR for now. If you'd like some ideas around new IA's there are some great ideas pending: https://duck.co/ideas/status/3

**Pull request closed without merging**